css/main.css

```diff
@@ -57,6 +57,21 @@ footer svg {
   text-shadow: 4px 4px 1px rgb(13 66 163), -3px -3px 1px rgb(148 186 255);
 }
 
+#welcome-splash-button {
```

Comment:
ty for styling the buttons!! much appreciated!!

Reply:
It is very basic so feel free to change it. :)